app/models/dish.rb

```diff
@@ -18,16 +18,24 @@ class Dish < ActiveRecord::Base
   RATE_DIMENSION_DISH = 'dish'
   ratyrate_rateable RATE_DIMENSION_DISH
 
-  def rates
+  # public activity
+  include PublicActivity::Model
+  tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: Proc.new{ |controller, model| model.try(:user) } 
```

Comment:
`Proc`s aren't fussy if they're given the wrong number of arguments, so a further simplification would be to leave out the unused `model` arg, so the line becomes:

```ruby
tracked owner: Proc.new{ | controller | controller.try(:current_user) }, recipient: :user 
```